app/model/config.py

```diff
@@ -19,11 +19,32 @@ class DocumentConfig(BaseModel):
     variants: Sequence[str]
 
 
+class CorpusData(BaseModel):
+    """Contains the Corpus and CorpusType info"""
+
+    corpus_import_id: str
+    title: str
+    description: str
+    corpus_type: str
+    corpus_type_description: str
+    taxonomy: TaxonomyData
```

Comment:
Hmm... we should probably define `TaxonomyData` in terms of db_client/models/dfce/taxonomy_entry.py

Reply:
This is being removed in the next PR anyway, but good point :)